tests/apps/animal/test_animal_api.py

```diff
@@ -25,6 +25,25 @@ def _test_animal_export(self, rewrite_data_files: bool, slug: str, key: int):
 
         assert data == json.loads(fn.read_text())
 
+    def _test_heatmap(self, rewrite_data_files: bool, slug: str, key: int, unpublished: bool):
```

Comment:
can you refactor `test_endpoint_export` so we dont have two helper functions that basically do the same thing?